buddyboss/buddyboss-platform
# PROD-759 Activity URL search with non ascii characters - issue fixed

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2809 .

### How to test the changes in this Pull Request:

1. Go to 'Buddyboss Activity' page
2. add the following URI with activity page URL: /?s=한글 or /?activity_search=한글
3. Then press enter
4. See its working now.

### Proof Screenshots or Video

https://www.loom.com/share/7cf1b848b72e4d3187d8382cd490bda6

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
